pkg/cover/backend/dwarf.go

```diff
@@ -201,6 +204,8 @@ func makeDWARFUnsafe(params *dwarfParams) (*Impl, error) {
 		allRanges = append(allRanges, result.ranges...)
 		allUnits = append(allUnits, result.units...)
 	}
+	log.Logf(0, "len(allUnits)==%v, len(allSymbols)=%v, len(allCoverPoints[0])=%v, len(allCoverPoints[1]=%v\n)",
```

Comment:
I don't think this is useful for end users.
Either remove or log at level 1.
Also it should be expressed in end user terms rather than as internal debugging mentioning variable names, e.g. "discovered %v source files...".
Also several inconsistencies: "==" vs "=", stray "\n)" at the end.